wasp/src/main/java/com/orhanobut/wasp/VolleyNetworkStack.java

```diff
@@ -46,9 +46,9 @@ private RequestQueue getRequestQueue() {
 
     private void addToQueue(final WaspRequest waspRequest, final CallBack callBack) {
         final String url = waspRequest.getUrl();
```

Comment:
I think we can also remove final from here, since we use url as parameter now, and would be better to remove the final from callback param. I put them to use in the anonymous class but no need now.